lib/src/controls/better_player_controls_state.dart

```diff
@@ -10,7 +10,8 @@ import 'package:flutter/material.dart';
 import '../core/better_player_utils.dart';
 
 ///Base class for both material and cupertino controls
-abstract class BetterPlayerControlsState<T extends StatefulWidget> extends State<T> {
+abstract class BetterPlayerControlsState<T extends StatefulWidget>
```

Comment:
na pewno masz 120 znaków ustawione zeby wrapować nową linię? wszystkie pliki praktycznie są zmienione - ciezko sie to czyta :/